src/__tests__/renderer-test.js

```diff
@@ -429,46 +525,55 @@ test("parses indented code blocks", () => {
     }
 `;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("parses hashtag", () => {
   const text = `this is a #hashtag example`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("parses hashtag ignoring dash", () => {
   const text = `dash should end #hashtag-dash`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("parses image", () => {
   const text = `![example](http://example.com/logo.png)`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("parses link", () => {
   const text = `[google](http://google.com)`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("parses link within mark", () => {
   const text = `**[google](http://google.com)**`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("parses link with encoded characters", () => {
   const text = `[kibana](https://example.com/app/kibana#/discover?_g=%28refreshInterval:%28%27$$hashKey%27:%27object:1596%27,display:%2710%20seconds%27,pause:!f,section:1,value:10000%29,time:%28from:now-15m,mode:quick,to:now%29%29&_a=%28columns:!%28metadata.step,message,metadata.attempt_f,metadata.tries_f,metadata.error_class,metadata.url%29,index:%27logs-%27,interval:auto,query:%28query_string:%28analyze_wildcard:!t,query:%27metadata.at:%20Stepper*%27%29%29,sort:!%28time,desc%29%29)`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("parses link with percent symbol", () => {
   const text = `[kibana](https://example.com/app/kibana#/visualize/edit/Requests-%)`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, true);
 });
 
 test("ignores empty link", () => {
   const text = `[empty]()`;
   expect(getNodes(text)).toMatchSnapshot();
+  assertSymmetry(text, false);
```

Comment:
How is it possible that both this statement and the previous one can pass the test?